src/compiler/scala/tools/nsc/typechecker/Typers.scala

```diff
@@ -10,12 +10,6 @@
  * additional information regarding copyright ownership.
  */
 
-// Added: Sat Oct 7 16:08:21 2006
-//todo: use inherited type info also for vars and values
-
-// Added: Thu Apr 12 18:23:58 2007
-//todo: disallow C#D in superclass
-//todo: treat :::= correctly
```

Comment:
Should check first whether these petroglyphs are protected under international treaty as cultural treasures.